sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -36,6 +37,10 @@
 from sklearn.utils._testing import assert_allclose
 from sklearn.utils._testing import assert_array_equal
 from sklearn.utils._testing import ignore_warnings
+from sklearn.utils import _IS_32BIT
+from sklearn.utils.validation import check_random_state
+from sklearn.ensemble._base import MAX_RAND_SEED
```

Comment:
Just use the `np.iinfo`

Reply:
No strong opinion but I think using `MAX_RAND_SEED` makes things more explicit

Reply:
This is a bit reverting https://github.com/scikit-learn/scikit-learn/pull/15960 which is the follow-up of this: https://github.com/scikit-learn/scikit-learn/pull/15933#discussion_r360308947

Reply:
ok then